client/src/blog/post.scss

```
@@ -1,135 +1,128 @@
 @use "../ui/vars" as *;
 
-.blog-container.post {
-  max-width: calc(48rem + 4rem);
-
-  + .section-newsletter h2 {
-    font: var(--type-heading-h3);
-    margin: 0;
-  }
-}
-
-.blog-container {
-  --author-gap: 1rem;
-  --avatar-size: 3rem;
-
-  .date-author,
-  .author {
-    align-content: flex-start;
-    align-items: center;
-    display: flex;
-    flex-wrap: wrap;
+.blog-post-container {
+  display: grid;
+  gap: 3rem;
+  grid-template-areas:
+    "post"
+    "newsletter";
+  width: 100%;
+  @media screen and (min-width: $screen-lg) {
```

nit: (surprised stylelint doesn't enforce this)

```suggestion

  @media screen and (min-width: $screen-lg) {
```